plugins/outputs/cloudwatch/cloudwatch.go

```diff
@@ -30,7 +30,7 @@ import (
 )
 
 const (
-	defaultMaxDatumsPerCall        = 20     // PutMetricData only supports up to 20 data metrics per call by default
+	defaultMaxDatumsPerCall        = 1000   // PutMetricData only supports up to 1000 data metrics per call by default
```

Comment:
If the `defaultMaxDatumsPerCall` is increased, I would assume the `defaultMaxValuesPerDatum` would change. Therefore, would consider removing the rules when applied with internal flag. For example these two files: [maxDatumsPerCall](https://github.com/aws/amazon-cloudwatch-agent/blob/master/translator/translate/metrics/ruleMaxDatumsPerCall.go)  and also [maxValumPerDatum](https://github.com/aws/amazon-cloudwatch-agent/blob/master/translator/translate/metrics/ruleMaxValuesPerDatum.go)